content/en/case-studies/gw-discov.md

```diff
@@ -55,81 +76,93 @@ made from warped spacetime.
+    Once the obstacles related to understanding Einstein’s equations well enough
+to solve them using supercomputers are taken care of, the next big challenge was
+making data comprehensible to the human brain. Simulation modeling as well as
+signal detection requires effective visualization techniques.  Visualization
+also plays a role in lending more credibility to numerical relativity in the
+eyes of pure science aficionados, who did not give enough importance to
+numerical relativity until imaging and simulations made it easier to comprehend
+results for a larger audience.  Speed of complex computations and rendering,
+re-rendering images and simulations using latest experimental inputs and
+insights can be a time consuming activity that challenges researchers in this
+domain.
+
+{{< figure src="/images/content_images/cs/gw_measurement.png" 
+    class="fig-center" alt="gravitational waves measurement" 
+    caption="**LIGO measurement of gravitational waves at Hanford and Livingston comparing signals to those expected due to a black hole merger event**" 
+    attr="(**Graph Credits:** LIGO Scientific Collboration & Virgo Collaboration" 
+    attrlink="https://physics.aps.org/featured-article-pdf/10.1103/PhysRevLett.116.061102" >}}
 
 ## NumPy’s Role in the detection of Gravitational Waves
 	
 Gravitational waves emitted from the merger cannot be computed using any
-technique except brute force numerical relativity using supercomputers.
-The amount of data LIGO collects is as incomprehensibly large as gravitational
-wave signals are small.
-
-NumPy, the standard numerical analysis package for Python,  was utilized by
-the software used for various tasks performed during the GW detection project
-at LIGO. NumPy helped in solving complex maths and data manipulation at high
-speed.  Here are some examples:
-
-* [Signal Processing](https://www.uv.es/virgogroup/Denoising_ROF.html): Glitch
-  detection,  [Noise identification and Data Characterization](https://ep2016.europython.eu/media/conference/slides/pyhton-in-gravitational-waves-research-communities.pdf)
-  (NumPy, scikit-learn, scipy, matplotlib, pandas, pyCharm )
+technique except brute force numerical relativity using supercomputers.  The
+amount of data LIGO collects is as incomprehensibly large and requires careful
+processing like any other data science problem.  Although it is technologically
+more challenging to achieve the requisite sensitivity for capturing
+gravitaional waves and is way harder than dealing with large scale data, yet
+the latter is complex enough to require numerical processing skills of several
+Python tools including
+[NumPy](https://towardsdatascience.com/2020-how-a-i-could-help-astronomers-sorting-big-data-811571705707).
```

Comment:
```suggestion
```
We should be careful mentioning NumPy in the context of the template matching and relativity calculations. I'm not sure that NumPy was used in those regimes.

My suggestion would be to delete this paragraph, the following paragraph on NumPy doesn't need any sort of segue, and I think the statements in this one are misleading and/or inaccurate.

I see that you have a link to a towardsdatascience article - maybe this section was included for the "honeypot" aspect. I must say though that the data/tutorials from PyCBC are much more compelling than the one in this article (which uses pre-processed data that doesn't really use NumPy except for loading the data).